### ISSUE TITLE
Prod Bugs Round 1

### ISSUE THREAD

**Issue by rickkdev** [OpenQDev/OpenQ-Frontend#1093]
Hey all, pushing to prod created a couple of bugs I'll make a list here:

- [x] Creating a contest bounty, when I try to "Connect Wallet", I can select a wallet but then I get redirected to the modal without anything happening (I can mint the bounty though if my wallet is already connected)
- [x] After the bounty gets minted I get the message "waiting for graph to index" 2 seconds later this error: 
![minted](https://user-images.githubusercontent.com/28826387/207958562-112baa3a-ce5f-4a7f-a9ee-e500e4ec4f79.png)
->  After 10 seconds I get redirected to main page where I can found the bounty I minted

**Comment by Christopher-Stevers** [comment]
Just tried minting on staging it seems to be working fine.

**Comment by Christopher-Stevers** [comment]
Looking into the connect wallet issue rn.

**Comment by FlacoJones** [comment]
Ya I'm pretty sure this is a race condition between the Graph indexing it, and the OZ Defender Autotask putting it in our API

I got the same - and on refresh it begins to work

In the console, there are a whole bunch of `bounty: null` returns

**Comment by FlacoJones** [comment]
Yeah confirmed, half the time i get no error, half the time i do

**Comment by FlacoJones** [comment]
Duplicate of https://github.com/OpenQDev/OpenQ-Frontend/issues/1084

**Comment by FlacoJones** [comment]
Perhaps we could, like we do with the `bountyRefresh`, add a non-null bounty return from API as well?

Or, like we do with the Graph now, we can show what we can from Github, and wait on the OpenQ-API if it returns null.
